src/main/java/dan200/computercraft/shared/pocket/apis/PocketAPI.java

```diff
@@ -78,11 +78,12 @@ public void shutdown()
                     {
                         if( !(m_computer.getEntity() instanceof EntityPlayer) )
                         {
-                            throw new LuaException( "Cannot find player" );
+                            return new Object[] { false, "Cannot find player" };
                         }
 
                         EntityPlayer player = (EntityPlayer) m_computer.getEntity();
                         InventoryPlayer inventory = player.inventory;
+                        if( !m_computer.inInventory() ) return new Object[] { false, "Not in an inventory" };
```

Comment:
How about finding the pocket computers not updated to set their private fields to `null` on `ServerTickEvent(Phase.END)`?